plugin/pkg/shoot/validator/admission_test.go

```diff
@@ -761,7 +761,7 @@ var _ = Describe("validator", func() {
 				err := admissionHandler.Admit(ctx, attrs, nil)
 
 				Expect(err).To(BeForbiddenError())
-				Expect(err).To(MatchError(ContainSubstring("spec.seedName cannot be changed by patching the shoot, please use the shoots/binding subresource")))
+				Expect(err).To(MatchError(ContainSubstring("spec.seedName 'seed' cannot be changed to 'nil' by patching the shoot, please use the shoots/binding subresource")))
```

Comment:
seedName cannot be set to nil once set, right? So maybe we print that it cannot be nil at all, even with the binding resource.

Reply:
I was wondering about that as well, but decided to leave that error for the `binding` subresource validation so as to keep the current "behaviour".

Otherwise I could potentially print two errors depending on whether the new `.spec.seedName` is nil or not:

When it is not nil:
```
spec.seedName 'seed-name' cannot be changed to 'new-seed' by patching the shoot, please use the shoots/binding subresource
```
When it is nil:
```
spec.seedName is already set to `seed-name` and cannot be changed to 'nil'
```

Reply:
I would prefer the two error approach.
In binding, we print similar error for the nil case, you could use this error there too.

Reply:
Done